tests/path.py

```diff
@@ -145,7 +145,7 @@ def text(self, encoding='utf-8', **kwargs):
         mode = 'rU' if PY2 else 'r'
         with open(self, mode=mode, encoding=encoding, **kwargs) as f:
             text = f.read()
-        contents = repr_as(text, '<%s contents>' % self.basename())
+        contents = repr_as(text, '<%s contents: %r>' % (self.basename(), text))
```

Comment:
Confirmed.
1) is not appeared with pytest: https://travis-ci.org/sphinx-doc/sphinx/jobs/188570306#L1829
2) is bad side-effect of suppressing (1).

> If pytest shows human-readable list of testcases, repr_as is no longer needed. So we should remove it.

OK. I'll remove it.